lib/openhab/core/types/hsb_type.rb

```diff
@@ -175,6 +188,13 @@ def to_s
         # @!method to_xy
         #   Convert to the xyY values representing this object's color in CIE XY color model
         #   @return [[PercentType, PercentType, PercentType]]
+
+        # @!attribute [r] cct
+        # @return [QuantityType] The color temperature in Kelvin
+        # @since openHAB 4.3
+        def cct
+          ColorUtil.xyToKelvin(to_xy[0..1].map { |x| x.double_value / 100 }) | "K"
```

Comment:
Curious as to why not `xy_to_kelvin`?

Reply:
cause I probably copy/pasted from ColorUtil. fixed.